webapp/app/assessment/components/dataTable/table/cell/cellOdp.js

```diff
@@ -24,19 +25,20 @@ const CellOdp = props => {
 
   const dispatch = useDispatch()
   const userInfo = useUserInfo()
-  const valid = useSelector(state => {
+  const valid = useSelector((state) => {
     if (!userInfo || !validator) {
       return true
     }
     return validator(datum)(state)
   })
+  const printView = useSelector(AppState.isPrintView)
```

Comment:
can we create hook?

const { printView, printingOnlyTables } = usePrintView()

Reply:
yes, but maybe returns an array? ```const [printView, printOnlyTablesView]```?

Reply:
Yes